ivy/functional/backends/numpy/experimental/linear_algebra.py

```diff
@@ -89,3 +89,12 @@ def kron(
 
 
 kron.support_native_out = False
+
+
+def matrix_exp(
```

Comment:
just a small tip, you should check out how to state the unsupported datatypes for these backend functions or find a way to convert them in the function.

Reply:
and I suppose there are ways to specify what datatype you what to generate for your test as well written in the deep dive document.